tracing-log/src/interest_cache.rs

```diff
@@ -235,7 +235,7 @@ mod tests {
 
     fn lock_for_test() -> impl Drop {
         // We need to make sure only one test runs at a time.
-        static LOCK: Lazy<Mutex<()>> = Lazy::new(Mutex::new);
+        static LOCK: Lazy<Mutex<()>> = Lazy::new(|| Mutex::new(()));
```

Comment:
why was this change necessary?

Reply:
For whatever reason, my rustc version didn't liked it.

Reply:
Should I revert this?

Reply:
Done